operator/src/main/resources/scripts/startServer.sh

```diff
@@ -137,6 +142,8 @@ function copySitCfg() {
   if [ $? = 0 ]; then
     for local_fname in ${src_dir}/${fil_prefix}*.xml ; do
       copyIfChanged $local_fname $tgt_dir/`basename ${local_fname/${fil_prefix}//}`
+      trace "Printing contents of situational configuration file $local_fname:"
+      echo `cat $local_fname`
```

Comment:
Why ```echo `cat $local_fname` ``` instead of just ``` cat $local_fname ```?

Reply:
agree. not sure why didn't do that earlier.